accounts/urls.py

```diff
@@ -1,13 +1,15 @@
-from django.urls import path, include
-from django.contrib import admin
 from . import views
-from django.contrib.auth.views import LoginView
+from django.contrib import admin
 from django.conf.urls import url
+from django.urls import path, include
+from django.contrib.auth.views import LoginView
+from django.contrib.auth import views as auth_views
+
+
 app_name='accounts'
 urlpatterns = [
-    path('register/', views.register, name='register'),
-    path('login/', LoginView.as_view(template_name='registration/login.html'), name='login'),
-    path('sent/', views.activation_sent_view, name="activation_sent"),
-    path('activate/<slug:uidb64>/<slug:token>/', views.activate, name='activate'),
-    path('profile/<int:user_id>/',views.profile, name='profile'),
-] 
+    path('profile/<int:pk>/', views.profile, name='profile'),
+    path('LOGOUT/', auth_views.LogoutView.as_view(), name='Logout'),
+    path('edit/<int:id>/', views.profile_edit, name='edit'),
+    path('Profile/', views.profile_account, name='profile1'),
```

Comment:
change 1st and 4th to profile/<int:pk> ...... name='profile' and profile/<int:pk> ...... name='profile_account'